frontend/src/metabase/components/SaveStatus/SaveStatus.styled.ts

```diff
@@ -0,0 +1,27 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+const SaveStatusBase = styled.div`
+  line-height: 1;
+  padding: 0.75rem 1rem;
+  border-radius: 0.5rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-weight: bold;
+`;
+
+export const SaveStatusLoading = styled(SaveStatusBase)`
+  color: ${color("text-medium")};
+  background: white;
+`;
+
+export const SaveStatusSuccess = styled(SaveStatusBase)`
+  color: white;
+  background: ${color("success")};
+`;
+
+export const SaveStatusError = styled(SaveStatusBase)`
+  color: white;
+  background: ${color("error")};
+`;
```

Comment:
these styles look a bit too buttony I think.

![Screen Shot 2024-02-15 at 12 44 18 PM](https://github.com/metabase/metabase/assets/30528226/7cf58ff2-79b4-4afa-ae2f-f755ede52cd4)
_(one of these is a button, one is not)_

maybe this could be a good opportunity to implement https://v6.mantine.dev/core/notification/ ?

Reply:
also.. why do we show notifications in the top right here, and _everywhere_ else they're in the bottom right?